src/app/resources/resource.js

```diff
@@ -0,0 +1,31 @@
+ (function() {
+    'use strict';
+
+    angular
+        .module('resource')
+        .factory('Resource', Resource);
+
+    function Resource($resource) {
+
+        return function(url, params, methods) {
+            var defaults = {
+                update: { method: 'put', isArray: false },
+                create: { method: 'post' }
+            };
+
+            methods = angular.extend(defaults, methods);
+
+            var resource = $resource(url, params, methods);
```

Comment:
Add `config.API_URL` here so that you don't have to repeat it in every actual resource